src/include/rados/rados_types.hpp

```diff
@@ -52,23 +52,23 @@ struct object_id_t {
 
 struct err_t {
   enum {
-    ATTR_UNEXPECTED      = 1 << 0,
     SHARD_MISSING        = 1 << 1,
     SHARD_STAT_ERR       = 1 << 2,
     SHARD_READ_ERR       = 1 << 3,
```

Comment:
Yes, we can reuse bits, except for user created client code using librados directly.  We are incompatible but I don't mind since we are making extensive changes.